### PR TITLE
feat- React: Basic Route Setup and Layout #40989

### PR DESCRIPTION
**Pull request by Priyankatiyar**
Create a new feature branch 40989.
[Refer Task](https://redmine.successive.tech/issues/40989)
NotFoundRoute
childrRout
inputRout
textFielRout
traineeRout
d-11LOgin

### CODE REVIEW

File: training/src/pages/ChildrenDemo/ChildrenDemo.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { ThemeProvider } from '@material-ui/core';
 import Math from '../../components/Math/Math';
 import Theme from '../../theme';
+import NavBar from '../../layouts/components/Navbar/Navbar';
```
Comment (shashank-nx) [inline training/src/pages/ChildrenDemo/ChildrenDemo.jsx]: Change` NavBar` Path with `'../../layouts/components/Navbar/'`

File: training/src/pages/InputDemo/InputDemo.jsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import {
   TextField, SelectField, RadioGroup, ButtonField,
 } from '../../components';
 import { Text } from '../../components/TextField/style';
+import NavBar from '../../layouts/components/Navbar/Navbar';
```
Comment (shashank-nx) [inline training/src/pages/InputDemo/InputDemo.jsx]: Change `NavBar` Path with `'../../layouts/components/Navbar/'`

File: training/src/pages/Login/Login.jsx [hunk 3]
```diff
@@ -6,16 +6,22 @@ import {
 } from '@material-ui/core';
 import { Email, VisibilityOff, LockOutlined } from '@material-ui/icons';
 import * as yup from 'yup';
+import Footer from '../../layouts/components/Footer/Footer';
```
Comment (shashank-nx) [inline training/src/pages/Login/Login.jsx]: Change `Footer` Path with `'../../layouts/components/Footer/'`

File: training/src/pages/NoMatch/NoMatch.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import NavBar from '../../layouts/components/Navbar/Navbar';
```
Comment (shashank-nx) [inline training/src/pages/NoMatch/NoMatch.jsx]: Change `NavBar` Path with `'../../layouts/components/Navbar/'`

File: training/src/pages/TextFieldDemo/TextFieldDemo.jsx [hunk 5]
```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import { TextField } from '../../components';
 import { Slider } from '../../components/slider';
+import NavBar from '../../layouts/components/Navbar/Navbar';
```
Comment (shashank-nx) [inline training/src/pages/TextFieldDemo/TextFieldDemo.jsx]: Change `NavBar` Path with `'../../layouts/components/Navbar/'`

File: training/src/pages/Trainee/Trainee.jsx [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import { Button } from '@material-ui/core';
 import React from 'react';
 import AddDialog from './components/AddDialog/AddDialog';
-import NavBar from '../components/Navbar/Navbar';
+import NavBar from '../../layouts/components/Navbar/Navbar';
```
Comment (shashank-nx) [inline training/src/pages/Trainee/Trainee.jsx]: Change `NavBar` Path with `'../../layouts/components/Navbar/'`

File: training/src/routes/AuthRoute.jsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import * as PropTypes from 'prop-types';
+import { Route } from 'react-router-dom';
+import AuthLayout from '../layouts/AuthLayout/AuthLayout';
```
Comment (shashank-nx) [inline training/src/routes/AuthRoute.jsx]: Change `AuthLayout` Path with `'../layouts/AuthLayout/AuthLayout'`

File: training/src/routes/PrivateRoute.jsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Route } from 'react-router-dom';
+import { PrivateLayout } from '../layouts/index';
```
Comment (shashank-nx) [inline training/src/routes/PrivateRoute.jsx]: import { PrivateLayout } from '../layouts/index'; No need to pass index here.

**Review by shashank-nx — CHANGES_REQUESTED** [review]

File: training/src/pages/ChildrenDemo/ChildrenDemo.jsx (hunk 1 above)
Comment (Priyankatiyar) [reply training/src/pages/ChildrenDemo/ChildrenDemo.jsx]: Done.

**Review by shashank-nx — APPROVED** [review]
Looks fine to me.

**Pull request merged**